### PR TITLE
Update Diagram and FileManager API descriptions

### PR DESCRIPTION
**Pull request by LanaDX**

### CODE REVIEW

File: api-reference/10 UI Components/dxDiagram/1 Configuration/onRequestLayoutUpdate.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ default: null
 ---
 ---
 ##### shortDescription
-A function that is executed after diagram data is reloaded and allows you to specify whether or not the UI component should update the diagram layout.
+A function that is executed after diagram data is reloaded and allows you to specify whether or not the UI component should reapply its auto layout.
```
Comment (arminal) [inline api-reference/10 UI Components/dxDiagram/1 Configuration/onRequestLayoutUpdate.md]:
```suggestion
A function that is executed after diagram data is reloaded. This function also allows you to specify whether or not the UI component should reapply its auto layout.
```

File: api-reference/10 UI Components/dxDiagram/1 Configuration/onRequestLayoutUpdate.md [hunk 2]
```diff
@@ -27,10 +27,10 @@ The UI component instance's name.
...
+The [autoLayout](/api-reference/10%20UI%20Components/dxDiagram/1%20Configuration/nodes/autoLayout/autoLayout.md '/Documentation/ApiReference/UI_Components/dxDiagram/Configuration/nodes/autoLayout/') option specifies an auto-layout algorithm that the UI component uses to build a diagram on data binding.
 
-Write the **onRequestLayoutUpdate** function to specify whether the UI component should recalculate and update the diagram layout after the data is reloaded.
+Write the **onRequestLayoutUpdate** function to specify whether the component should reapply its auto layout when a user changes diagram data by the UI or you modify the data source directly via [push services](/concepts/70%20Data%20Binding/5%20Data%20Layer/3%20Data%20Modification/30%20Integration%20with%20Push%20Services.md '/Documentation/Guide/Data_Binding/Data_Layer/#Data_Modification/Integration_with_Push_Services').
```
Comment (arminal) [inline api-reference/10 UI Components/dxDiagram/1 Configuration/onRequestLayoutUpdate.md]: Run on sentence. Please rewrite into 2 sentences.
```suggestion
Write the **onRequestLayoutUpdate** function to specify whether the component should reapply its auto layout when a user changes diagram data by [by?] the UI or you modify the data source directly via [push services](/concepts/70%20Data%20Binding/5%20Data%20Layer/3%20Data%20Modification/30%20Integration%20with%20Push%20Services.md '/Documentation/Guide/Data_Binding/Data_Layer/#Data_Modification/Integration_with_Push_Services').
```

File: api-reference/10 UI Components/dxDiagram/4 Events/requestLayoutUpdate.md [hunk 3]
```diff
@@ -4,7 +4,7 @@ type: eventType
 ---
 ---
 ##### shortDescription
-Raised after diagram data has been changed and requests whether the UI component should update the diagram layout.
+Raised after diagram data has been changed and requests whether the UI component should reapply its auto layout.
```
Comment (arminal) [inline api-reference/10 UI Components/dxDiagram/4 Events/requestLayoutUpdate.md]:
```suggestion
Raised after diagram data has been changed and sends a request to the UI component to reapply its auto layout.
```

File: api-reference/50 Common/Object Structures/UploadInfo/customData.md [hunk 4]
```diff
@@ -4,6 +4,36 @@ type: any
 ---
 ---
 ##### shortDescription
-Information saved during the file upload.
+Custom information that you can pass during the file upload. For instance, you can specify a custom file id when the first chunk of a specific file is being uploaded.
```
Comment (arminal) [inline api-reference/50 Common/Object Structures/UploadInfo/customData.md]:
```suggestion
Custom information that you can pass during file upload. For instance, you can specify a custom file ID when the first part of a file is being uploaded.
```

File: api-reference/10 UI Components/dxDiagram/1 Configuration/onRequestLayoutUpdate.md (hunk 1 above)
Comment (LanaDX) [reply api-reference/10 UI Components/dxDiagram/1 Configuration/onRequestLayoutUpdate.md]: Can we rewrite it in the following way?
```suggestion
A function that allows you to specify whether or not the UI component should reapply its auto layout after diagram data is reloaded.
```

File: api-reference/10 UI Components/dxDiagram/1 Configuration/onRequestLayoutUpdate.md (hunk 2 above)
Comment (LanaDX) [reply api-reference/10 UI Components/dxDiagram/1 Configuration/onRequestLayoutUpdate.md]:
```suggestion
Write the **onRequestLayoutUpdate** function to specify whether the component should reapply its auto layout when a user changes diagram data in the UI or you modify the data source directly via [push services](/concepts/70%20Data%20Binding/5%20Data%20Layer/3%20Data%20Modification/30%20Integration%20with%20Push%20Services.md '/Documentation/Guide/Data_Binding/Data_Layer/#Data_Modification/Integration_with_Push_Services').
```

File: api-reference/10 UI Components/dxDiagram/4 Events/requestLayoutUpdate.md (hunk 3 above)
Comment (LanaDX) [reply api-reference/10 UI Components/dxDiagram/4 Events/requestLayoutUpdate.md]: I think, the meaning has changed. Can I write it in the following way?
```suggestion
Raised after diagram data has been changed. Requests information whether the UI component should reapply its auto layout.
```

File: api-reference/10 UI Components/dxDiagram/1 Configuration/onRequestLayoutUpdate.md (hunk 1 above)
Comment (arminal) [reply api-reference/10 UI Components/dxDiagram/1 Configuration/onRequestLayoutUpdate.md]: ok

File: api-reference/10 UI Components/dxDiagram/1 Configuration/onRequestLayoutUpdate.md (hunk 2 above)
Comment (arminal) [reply api-reference/10 UI Components/dxDiagram/1 Configuration/onRequestLayoutUpdate.md]: ok

File: api-reference/10 UI Components/dxDiagram/4 Events/requestLayoutUpdate.md (hunk 3 above)
Comment (arminal) [reply api-reference/10 UI Components/dxDiagram/4 Events/requestLayoutUpdate.md]: Requests information whether the UI component should reapply its auto layout. - this doesn't work, can you say it differently? "requests whether should reapply" we can't keep that phrasing

Comment (LanaDX) [reply api-reference/10 UI Components/dxDiagram/4 Events/requestLayoutUpdate.md]:
```suggestion
Raised after diagram data has been changed.
```

**Review by vladaskorohodova — APPROVED** [review]

**Pull request merged**